src/OpenConext/UserLifecycle/Domain/Client/InformationResponseFactoryInterface.php

```diff
@@ -23,15 +25,18 @@
 interface InformationResponseFactoryInterface
 {
     /**
-     * @param array $response
      * @return InformationResponseInterface
      */
-    public function fromApiResponse(array $response);
+    public function fromApiResponse(
+        array $response,
+    );
 
     /**
-     * @param Exception $e
      * @param string $applicationName
      * @return InformationResponseInterface
      */
-    public function fromException(Exception $e, $applicationName);
+    public function fromException(
+        Exception $e,
```

Comment:
Did this not result in inspection errors? Should be Throwable right?

Reply:
Did not happen to me, I changed the formatting only. Since when should this be Throwable instead of Exception?

Reply:
That became the new standard in PHP 8 right? Given this is our own userland code. I can see we did not change this. Feel free to leave this as it is.